src/Bridges/DoctrineOrm/StructureDiffGenerator.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
+ * @link       https://github.com/nextras/migrations
+ */
+
+namespace Nextras\Migrations\Bridges\DoctrineOrm;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\SchemaTool;
+use Nextras;
+use Nextras\Migrations\IDiffGenerator;
+
+
+class StructureDiffGenerator implements IDiffGenerator
+{
+
+	/** @var EntityManager */
+	private $em;
+
+	/** @var string|NULL */
+	private $ignoredQueriesFile;
+
+
+	/**
+	 * @param EntityManager $em
+	 * @param string|NULL   $ignoredQueriesFile
+	 */
+	public function __construct(EntityManager $em, $ignoredQueriesFile)
+	{
+		$this->em = $em;
+		$this->ignoredQueriesFile = $ignoredQueriesFile;
+	}
+
+
+	/**
+	 * @return string
+	 */
+	public function getExtension()
```

Comment:
lepší by bylo `getFileExtension`, ale i tak, k čemu to je? Bylo by dobré dát to do doc